lib/entities/fiat_currency.dart

```diff
@@ -9,7 +9,7 @@ class FiatCurrency extends EnumerableItem<String> with Serializable<String> {
   static List<FiatCurrency> get all => _all.values.toList();
 
   static List<FiatCurrency> get currenciesAvailableToBuyWith =>
-      [aud, brl, cad, chf, czk, eur, dkk, gbp, hkd, ils, jpy, krw, mxn, myr, nok, nzd, pln, sek, sgd, thb, usd, zar];
+      [aud, brl, cad, chf, czk, eur, dkk, gbp, hkd, ils, jpy, krw, mxn, myr, nok, nzd, pln, sek, sgd, thb, usd, zar, idr];
```

Comment:
please revert this change, only specific currencies are available to buy with, if you wanna add Indonesian currency, please add it in a separate PR with a separate ticket

Reply:
also please fix conflicts with main